app/templates/check_results.html

```diff
@@ -259,6 +259,55 @@ <h3>
                             </table>
                         </div>
                     </div>
+
+                    <div class="card p-1 bg-light border rounded-3 mb-3">
```

Comment:
if there are multiple MXs the section is long:

![image](https://github.com/CERT-Polska/mailgoose/assets/1233067/887b0ee6-fb47-4cb5-abc0-74a374b0f14f)



is there a way to make it more concise?